app/javascript/dashboard/composables/useKeyboardEvents.js

```diff
@@ -0,0 +1,79 @@
+import { onMounted, onBeforeUnmount } from 'vue';
+import {
+  isActiveElementTypeable,
+  isEscape,
+} from 'shared/helpers/KeyboardHelpers';
+import { createKeybindingsHandler } from 'tinykeys';
+
+export function useKeyboardEvents(elRef, keyboardEvents) {
```

Comment:
Lets make the events first property and `elRef` optional. 

```suggestion
export function useKeyboardEvents(keyboardEvents, elRef = null) {
```

Later in the composable, we can use the root of the component as default, see [$el](https://vuejs.org/api/component-instance.html#el)